ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -96,6 +98,7 @@
                 time);
         stores.put(replica.getPartitionId(), store);
         partitionToReplicaMap.put(replica.getPartitionId(), replica);
+        expectedDirs.add(replica.getReplicaPath());
```

Comment:
Not related to this code change, but I am thinking if a new replica is successfully added on this disk and not yet get reflected in clustermap (actually it is not even started), we don't need to report extra directory created for this new replica, right?